mvg/mvg.py

```diff
@@ -56,7 +56,7 @@ def __init__(self, endpoint: str, token: str):
         self.endpoint = endpoint
         self.token = token
 
-        self.mvg_version = self.parse_version("v0.13.2")
+        self.mvg_version = self.parse_version("v0.14.0")
```

Comment:
```suggestion
        self.mvg_version = self.parse_version("v0.13.3")
```

as long as we are in version `0.x.y` we treat `x` as major version and  `y` as minor and bug fix. since adding a new method is not backward-incompatible, we can keep `x` and bump `y`

Reply:
Good to know that.